Dockerfile.dev

```diff
@@ -0,0 +1,35 @@
+FROM node:13.10.1-slim
+
+RUN apt-get update
+RUN apt-get install -y yarn
+RUN apt-get install -y git
+RUN apt-get install -y libacl1-dev
+RUN apt-get install -y libncurses-dev
+
+#esy install
+RUN apt-get install -y curl
+
+#esy bootstrap
+RUN apt-get install -y clang
+RUN apt-get install -y make
+RUN apt-get install -y m4
+RUN apt-get install -y nasm
+RUN apt-get install -y libfontconfig1-dev
+RUN apt-get install -y libx11-dev
+RUN apt-get install -y libsdl2-dev
+RUN apt-get install -y libbz2-dev
+RUN apt-get install -y libgtk-3-dev
+
+RUN apt-get install -y opam
```

Comment:
All `apt` operations should be put in a single `RUN`, see: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Something like:
```dockerfile
RUN apt-get update -q && apt-get install -y --no-install-recommends \
    yarn \
    git \
    …
    …
    …
 && rm -rf /var/lib/apt/lists/*
```

At the end the apt cache is no longer required and deleted, for a slimmer container image.

Same with the rest of the similar `RUN` instructions.